### PR TITLE
Always enforce bounds on brightness

### PR DESCRIPTION
**Pull request by jwunderl**
re: Peli's comment on https://github.com/microsoft/pxt-common-packages/pull

### CODE REVIEW

File: libs/screen/image.ts [hunk 1]
```diff
@@ -273,8 +273,10 @@ namespace helpers {
     function _setScreenBrightness(brightness: number) { }
 
     export function setScreenBrightness(img: Image, b: number) {
-        _helpers_workaround.brightness = b
-        _setScreenBrightness(_helpers_workaround.brightness)
+        if (b >= 10 && b <= 100) {
```
Comment (pelikhan) [inline libs/screen/image.ts]: Can you do clamp(10,100,...) instead?

**Comment by pelikhan** [comment]
Maybe also add | 0 for sanity

File: libs/screen/image.ts (hunk 1 above)
Comment (jwunderl) [reply libs/screen/image.ts]: I think I'll go through the math functions in a few and rename the parameters in a bit so they're nice and useable, as it's annoying to remember the order for clamp (I've seen helpers elsewhere with it set as ``clamp(min, num, max)`` as well)

**Comment by pelikhan** [comment]
Any change of variable order would be a breaking change...

________________________________

**Comment by jwunderl** [comment]
Not change the order, just ``min, max, input`` instead of ``a, b, c`` so the tooltips tell you what to do

**Pull request merged**